test/Gen.hs

```diff
@@ -130,15 +147,68 @@ w = genT
 
 type W = T "W"
 
+unit :: Gen ()
+unit = atom "()" ()
+
+identity :: Gen (a -> Identity a)
+identity = atom "Identity" Identity
+
 fn :: (Fn.Arg a, Fn.Vary a, Show a) => Gen b -> Gen (a -> b)
 fn b = Gen (lift (fmap (fmap runTerm) . showingFn <$> Fn.fn (fst <$> runWriterT (runGen b))))
 
+termFn :: Gen b -> Gen (a -> b)
+termFn b = Gen $ recursive Hedgehog.Gen.choice
+  [ runGen (atom "const" const <*> b) ]
+  []
```

Comment:
This is… a little weird. I _think_ `const x` is fine as a guaranteed-terminating function for the `MonadFix` laws because of parametricity—i.e. so long as the function doesn’t strictly evaluate its argument, one should be as good as any other here for the purposes of the `MonadFix` laws. I’m having trouble working out the proof, tho, so I left this phrased in `recursive` so that it’s trivial to add other, larger cases in future if we want to.